handler/rfc8628/auth_handler.go

```diff
@@ -0,0 +1,55 @@
+// Copyright © 2023 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+package rfc8628
+
+import (
+	"context"
+	"time"
+
+	"github.com/ory/fosite"
+	"github.com/ory/x/errorsx"
```

Comment:
imports need to be sorted. `github.com/ory/x/errorsx` belongs in a separate block above as it is imported from a separate project